src/proto/robot_status_msg.proto

```diff
@@ -29,6 +27,12 @@ message ThunderloopStatus
     uint64 iteration_time_ns                = 5;
 }
 
+message JetsonStatus
+{
+    // Jetson CPU temperature
+    double cpu_temperature = 3;
```

Comment:
the nano has multiple cpu cores, will we be using the temperature of one core to represent all?